pkg/controller/jaeger/jaeger_controller.go

```diff
@@ -358,5 +354,11 @@ func (r *ReconcileJaeger) apply(ctx context.Context, jaeger v1.Jaeger, str strat
 		return jaeger, nil
 	}
 
+	// we apply the daemonsets after everything else, to increase the changes of having services and deployments
```

Comment:
I'm guessing you mean "increase the chances"

Reply:
Fixed!